### DISCUSSION TITLE
Cyclic Spinning Wheel

### DISCUSSION THREAD

**Discussion by ghost** [community/community#34640] [New to GitHub]
I recently made a new GitHub account, and I am trying to deploy an app on Cyclic for school, but when I sign into Cyclic or try to deploy a starter the Cyclic page never loads. I had a different github account connected to Cyclic before.

**Comment by queenofcorgis** [comment]
Hi @user,

Thanks for being a part of the GitHub Community! We're glad you're here.

I wanted to jump in and mention that if you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Cyclic. It's definitely possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!

**Comment by jfullstackdev** [comment]
I'm not very sure if this is the app you are referring to, because if it is, we are fortunate it's open source and you can see the developers

https://github.com/cyclic-software

you might want to look repos related to your question, create an issue there to ask those and hopefully they will respond

**Reply by queenofcorgis** [reply]
Adding on to say the best way to contact the maintainers of that software is to:

- Check the README for instructions on how to operate the software and pointers to documentation or troubleshooting info.
- Check the [SUPPORT file,](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-support-resources-to-your-project) if one exists, for instructions on how to best contact the maintainers for support.
- Check the [CONTRIBUTING guide](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors), if one exists. Sometimes if there isn't a SUPPORT file, the CONTRIBUTING guide will give instructions on how to contact the maintainers for support.

All of these documents can be found, if they exist, in the repository (if that is correct) where you found the software itself.
